### PR TITLE
Bar Remap

### PR DESCRIPTION
**Pull request by mrhugo13**
This is my first map PR so if I fuck something up, please do point out.

## About The Pull Request
Remaps the Bar to fit to the aesthetic of the server

## Why It's Good For The Game
As said above, this Remap has the intention of making the bar fit the style/aesthetic of the server, instead of continuing it being what it currently is.

## Changelog
:cl: Vondiech
add: Remapped the bar
/:cl:

### CODE REVIEW

**Comment by mrhugo13** [comment]
Just to note, my intention was to use sofa's instead of winged chairs, however for some reason the sofa's don't have a back sprite.

**Comment by dapnee** [comment]
Mind posting an image? Also in mapping, if you need directions on a sprite, you right click it in the instance box and select "create instances from directions".

**Comment by mrhugo13** [comment]
> 
> 
> Mind posting an image? Also in mapping, if you need directions on a sprite, you right click it in the instance box and select "create instances from directions".

1st: thanks for telling me for the directions thing.
2nd: of course.
sure

**Comment by mrhugo13** [comment]
moved a few things around
moved a few things around

**Comment by dapnee** [comment]
Maybe make the wall on the bottom left into a glass+grille spawned so you can see into it, change the plasteel to white to make the area pop a bit.

**Comment by mrhugo13** [comment]
> 
> 
> Maybe make the wall on the bottom left into a glass+grille spawned so you can see into it, change the plasteel to white to make the area pop a bit.

Where in specific are you talking about?

**Review by niceping — CHANGES_REQUESTED** [review]
http://prntscr.com/mvvfnx
tile decal in walls, remove them

**Comment by mrhugo13** [comment]
> 
> 
> http://prntscr.com/mvvfnx
> tile decal in walls, remove them

**Comment by mrhugo13** [comment]
> 
> 
> http://prntscr.com/mvvfnx
> tile decal in walls, remove them

1: Woops wrong button
2: Fixed

**Comment by dapnee** [comment]
http://puu.sh/CXV0Z/9b591d40e0.png
Windows facing in toward high traffic areas makes them more interesting and the white scribbles are where I'd place white plasteel under the bar red decals

File: _maps/map_files/NSS-Sunset/NSS-Sunset.dmm [hunk 1]
```diff
@@ -27194,12 +27202,10 @@
...
 /obj/item/gun/ballistic/shotgun/doublebarrel,
-/obj/machinery/camera{
-	c_tag = "Bar - Backroom";
-	dir = 2
-	},
-/obj/structure/sink/kitchen{
-	pixel_y = 28
+/obj/machinery/requests_console{
+	department = "Security";
+	departmentType = 5;
```
Comment (dapnee) [inline _maps/map_files/NSS-Sunset/NSS-Sunset.dmm:27207]: Wrong request console
```suggestion
	department="Bar"
	departmentType = 2;
```

File: _maps/map_files/NSS-Sunset/NSS-Sunset.dmm [hunk 2]
```diff
@@ -27209,40 +27215,27 @@
...
+/obj/item/radio/intercom{
 	dir = 8;
-	freq = 1400;
-	location = "Bar"
-	},
-/obj/structure/plasticflaps/opaque,
-/obj/effect/turf_decal/bot{
-	dir = 1
+	freerange = 1;
+	name = "Station Intercom (Telecomms)";
```
Comment (dapnee) [inline _maps/map_files/NSS-Sunset/NSS-Sunset.dmm:27222]: Rename intercom

File: _maps/map_files/NSS-Sunset/NSS-Sunset.dmm [hunk 3]
```diff
@@ -28401,52 +28410,38 @@
...
 	dir = 4
 	},
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{
 	dir = 4
 	},
-/turf/open/floor/plating,
-/area/maintenance/starboard)
+/obj/machinery/light/small,
+/obj/machinery/camera{
+	c_tag = "Central Primary Hallway - Aft-Starboard";
```
Comment (dapnee) [inline _maps/map_files/NSS-Sunset/NSS-Sunset.dmm:28440]: c_tag needs to reflect where the camera is, so something like
```suggestion
	c_tag = "Bar Backroom";
```
cameras have a 7 tile radius, so I'd suggest making it one camera in the back above the vendor

**Review by dapnee — CHANGES_REQUESTED** [review]
A few changes I feel are necessary.

**Comment by mrhugo13** [comment]
@dapnee Changes that you felt are necessary are now included, aswell as the glass window thing being included, however you will have to explain the"White plasteel" thing to me in the discord.

**Comment by mrhugo13** [comment]
looking_currently
Just wanna note how it currently looks

**Pull request merged**